db/models/topics.models.js

```diff
@@ -41,9 +42,26 @@ function fetchCommentsByArticleId(articleId) {
 		});
 }
 
+function newComment(articleId, commentBody) {
+	const newCommentValuesArray = [
+		commentBody.body,
+		articleId,
+		commentBody.username,
+	];
+
+	const formattedComments = format(
+		"INSERT INTO comments(body, article_id, author) VALUES %L RETURNING *",
+		[newCommentValuesArray]
+	);
```

Comment:
There's no need to use pg format. This is useful when you are inserting multiple comments. In your case, it is just 1 comment